src/Components/Header/Header.js

```diff
@@ -24,10 +24,18 @@ const Header = ({ togView, currentUser }) => {
   
   const selUser = JSON.parse(sessionStorage.getItem('SEuserBtnStatus')) || 'customer'
```

Comment:
great work around Is SEuserBtnStatus a session storage item or something you created?